frontend/src/App.js

```diff
@@ -21,6 +24,11 @@ const App = () => {
         <Route path="/addQuestion" element={<AddQuestionPage />} />
         <Route path="/selectQuestion" element={<SelectQuestionPage />} />
         <Route path="/editQuestion" element={<EditQuestionPage />} />
+        <Route path="/selectlanguage" element={<SelectLanguage />} />
+        <Route path="/questionaire" element={<ClientDataForm />} />
+        <Route path="/table" element={<ClientDataTable />} />
+        <Route path="/adminpanel" element={<AdminPanel />} />
```

Comment:
Remove `<Route path="/adminpanel" element={<AdminPanel />} />` if `AdminPangel.js` is not merged.